app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  validates :comment_text, presence: true
```

Comment:
- [ ] Kindly add validation to check if the input is not empty. For example, a comment like '' should not be valid.